examples/with-apollo/lib/with-apollo-client.js

```diff
@@ -20,14 +20,7 @@ export default App => {
       if (typeof window === 'undefined') {
         try {
           // Run all GraphQL queries
-          await getDataFromTree(
-            <App
-              {...appProps}
-              Component={Component}
-              router={router}
-              apolloClient={apollo}
-            />
-          )
+          await getDataFromTree(<AppTree apolloClient={apollo} {...appProps} />)
```

Comment:
I think we should update the examples after it's on stable otherwise it'll break.

Reply:
👌 removed for now